src/main/resources/hudson/security/table.js

```diff
@@ -40,15 +40,15 @@ Behaviour.specify(".matrix-auth-add-button", 'GlobalMatrixAuthorizationStrategy'
 
     var tooltipAttributeName = getTooltipAttributeName();
 
-    findElementsBySelector(copy, ".stop a").each(function(item) {
+    findElementsBySelector(copy, ".stop a").forEach(function(item) {
```

Comment:
Tooltips for actions in newly added rows don't have placeholders, but proper labels.